src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/commands.py

```diff
@@ -37,6 +37,19 @@ def transform_ip_addresses(result):
     return transformed
 
 
+def transform_vm(result):
+    return OrderedDict([('name', result['name']),
```

Comment:
Are we fine with explicitly dropping ProvisioningState and VmId from the current output? (I suspect we are)